src/crypto/sign/ed25519.rs

```diff
@@ -149,6 +152,35 @@ pub fn verify_detached(
     }
 }
 
+/// Converts Ed25519 keypair to Curve25519 keyipair.
+pub fn convert_ed_keys_to_curve25519(
+    pk: PublicKey,
+    sk: SecretKey
+) -> ([u8; SCALARMULTBYTES], [u8; SCALARMULTBYTES]) {
+    let mut pk = clone_into_array(&pk[..]);
```

Comment:
Should `pk` and `sk` be as mutable ? As I can see `crypto_sign_ed25519_sk_to_curve25519` accepts constant pointer to array ?

Reply:
You are right, they shouldn't.